docs/man_pages/lib-management/library-add.md

```diff
@@ -7,21 +7,23 @@ General | `$ tns library add <Platform> <Library Path>`
 
 Adds a locally stored native library to the current project. <% if(isHtml) { %>Copies the library files to the `lib/<platform>` folder in your project and configures the platform-specific projects in `platforms/<platform>` to work with the library. Build operations might perform additional configuration changes on the platform-specific project in `platforms/<platform>`.<% } %>
 
+IMPORTANT: The `tns library add` command is deprecated and will be removed in a future release. Use the plugin set of commands instead. For more information, run `tns help plugin`.
```

Comment:
Also, it seems that we're probably deprecating the entire library set of commands. Shouldn't we apply this message to the library command as well? If we decide to do this Make the message "The `tns library` set of commands is deprecated and..."